jax/api.py

```diff
@@ -23,13 +23,14 @@
 arrays.
 """
 
-
+# TODO(vanderplas): define exactly what attributes we want in jax.api.
```

Comment:
Note that `api` isn't a public name; the entire thing should be hidden in an underscored part of `jax`. There is already a shim that hides most of the symbols in `api`.

Reply:
got it, removed.